ci/cron/src/Main.hs

```diff
@@ -203,13 +203,29 @@ build_docs_folder path versions latest = do
             shell_ $ "cp -r " <> from <> " " <> to
         build version path = do
             shell_ $ "git checkout v" <> version
+            shell_ "git reset --hard"
             -- Maven does not accept http connections anymore; this patches the
             -- scala rules for Bazel to use https instead. This is not needed
             -- after 0.13.43.
             if to_v version < to_v "0.13.44"
             then do
                 shell_ "git -c user.name=CI -c user.email=CI@example.com cherry-pick 0c4f9d7f92c4f2f7e2a75a0d85db02e20cbb497b"
             else pure ()
+            -- Starting with 0.13.54, we have changed the way in which we
+            -- trigger releases. Rather than releasing the current commit by
+            -- changing the VERSION file, we now mark an existing commit as the
+            -- source code for a release by changing the LATEST file. However,
+            -- release notes still need to be taken from the release commit
+            -- (i.e. the one that changes the LATEST file, not the one being
+            -- pointed to).
+            if to_v version > to_v "0.13.53"
+            then do
+                -- The release-triggering commit does not have a tag, so we
+                -- need to find it by walking through the git history of the
+                -- LATEST file.
+                sha <- find_commit_for_version version
+                shell_ $ "git checkout " <> sha <> " -- docs/source/support/release-notes.rst"
```

Comment:
The idea is that, when making the release, you have taken the release notes up to the source commit and put that in the release commit. So that should work out, as the release notes in the release commit are really as of the source commit and therefore cannot have references to things that came out after it.

Reply:
I get the intention what I’m a bit worried about is that this is not what `bazel build //docs` checks. But I’m fine with this for now.

Reply:
I'm happy to consider any suggestion on how to improve this, but I can't see any way out of it myself.